index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <meta charset="utf-8">
```

Comment:
To make sure your responsiveness works while in the chrome inspector, add the following tag to your html:

`<meta name="viewport" content="width=device-width,initial-scale=1">`